sr_unix/ydb_env_set.gtc

```diff
@@ -27,6 +27,12 @@ if [ ! -z "$ydb_chset" ] ; then export ydb_tmp_chset=$ydb_chset
 elif [ ! -z "$gtm_chset " ] ; then export ydb_tmp_chset=$gtm_chset
 fi
 
+# Save ydb_routines / gtmroutines if defined because
+# this script will overwrite ydb_routines if defined
+# to make sure there is a path to the required routine
+if [ ! -z "$ydb_routines" ] ; then ydb_tmp_routines="$ydb_routines" fi
+if [ ! -z "$gtmroutines" ] ; then ydb_tmp_routines="$gtmroutines" fi
```

Comment:
I'm not understanding how saving both ydb_routines and gtmroutines into the same save var (ydb_tmp_routines) preserves the values of both "unchanged". It is possible for them to neither, either, or both be set and to have different values - and in the latter case ydb_routines should override gtmroutines but the above doesn't seem to keep the values separate nor to allow ydb_routines to override.

Reply:
I was mimicking what was done with gtm_passwd/ydb_passwd. It saves it into the same variable. I can flip the statements so that ydb_routines takes precedence.